daml/Tests/Distribution/Auction.daml

```diff
@@ -260,8 +260,11 @@ assertAssetDeposits auction issuer originalDeposits price quantity bid = do
   quotedDeposit <- queryFilter @AssetDeposit bid.customer (\deposit -> deposit.account == bid.tradingAccount && deposit.asset.id == auction.quotedAssetId)
   case quotedDeposit of
     ([(_, x)]) -> do
-      let originalDeposit = filter (\deposit -> deposit.account.owner == x.account.owner) originalDeposits
-      assertMsg "Investor/Bidder split deposit contains incorrect quantity" (x.asset.quantity == (bid.details.quantity * price))
+      let originalDeposit = head $ filter (\deposit -> deposit.account.owner == x.account.owner) originalDeposits
+      let newQuantity = case bid.status of
+            (Bidding.PartialAllocation _ quantity) -> quantity
+            _                                      -> bid.details.quantity
+      assertMsg "Investor/Bidder split deposit contains incorrect quantity" (x.asset.quantity == (originalDeposit.asset.quantity - (newQuantity * price)))
```

Comment:
this previously wasn't getting called due to the asset id's for USD having different signatories, now that it is getting called, this fixes the assert.